src/nni_manager/rest_server/restValidationSchemas.ts

```diff
@@ -103,7 +103,6 @@ export namespace ValidationSchemas {
             }),
             pai_config: joi.object({ // eslint-disable-line @typescript-eslint/camelcase
                 userName: joi.string().min(1).required(),
-                passWord: joi.string().min(1),
```

Comment:
please also update doc accordingly

Reply:
the PAIMode doc has already removed the password configuration, while it keeps in our code. Only need to remove code this time.